tests/test_imaging.py

```diff
@@ -1,9 +1,16 @@
 import shutil
+import sys
 import tempfile
 
+import pytest
+
 from constructor.imaging import write_images
 
 
+@pytest.mark.skipif(
+    sys.platform != 'win32' and sys.platform != 'darwin',
```

Comment:
Why not?

```suggestion
    sys.platform == "linux",
```

Reply:
I did it because `sys.platform` can have more values than "win32", "darwin", and "linux". I don't mind changing it if you think this is better.

I did adjust the wording for the reason though.